modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-a-file.adoc

```diff
@@ -86,5 +86,5 @@ components:
        containerPath: /my/new/path
    ...
 ----
-+
-Note that for this kind of overrides, components must declare an alias to be able to distinguish containers which belong to them and apply override path exclusively for those containers.
+
+NOTE: For such overrides, you must declare an alias for the components to distinguish the containers which belong to them and to apply the override path exclusively to those containers.
```

Comment:
I'm not sure which overrides you're referring to. Please be specific. This is a run-on sentence. It's meaning is hard to decipher. In this case, the meaning of "them" is ambiguous. The `to...to...to...to` construction is hard to interpret. Please break this into multiple simple statements.

Reply:
I get it. It's only two infinitives in parallel, but those are difficult to follow, so need to improve the readability here. I'll see how I can split this into two sentences and possibly rearrange them as well.

Reply:
There are other issues here, which we'll discuss after you make the updates.